config/proxies.ini

```diff
@@ -0,0 +1,61 @@
+# For configuration instructions, please see:
+# http://opensciencegrid.github.io/docs/other/install-gwms-frontend/
+
+##################
+# FRONTEND PROXY #
+##################
+
+# There must only one be one [FRONTEND] section
+
+[FRONTEND]
+
+# Paths to frontend certificate and key
+proxy_cert = /etc/grid-security/gwms-frontend/frontendcert.pem
+proxy_key = /etc/grid-security/gwms-frontend/frontendkey.pem
+
+# Path to the output proxy
+# This should match the path specified in frontend.xml
+output = /etc/grid-security/gwms-frontend/frontend.proxy
+
+###############
+# PILOT PROXY #
+###############
+
+# For each pilot proxy, add a new proxy section.
+# Each pilot proxy section must start with the word "PILOT"
+# and have a unique "NAME"
+
+[PILOT NAME]
+
+# Paths to the pilot certificate and key
+proxy_cert = /etc/grid-security/gwms-frontend/pilotcert.pem
+proxy_key = /etc/grid-security/gwms-frontend/pilotkey.pem
+
+##################################
+# Virtual Organization information
+# vo:
+#     name of the virtual organization
+# fqan (optional, default: /Role=NULL/Capability=NULL):
+#     Request FQAN VO attribute
+# use_voms_admin (optional, default: false):
+#     If true, contact the VO's VOMS Admin server to sign the VOMS attributes
```

Comment:
This attribute is misnamed -- technically, it's the "VOMS server" (which, in 2.x, is written in C) while VOMS Admin (written in Java) is the web frontend.

Might as well get it correct to start out with...